docs/src/modules/branding/AppSearch.tsx

```diff
@@ -163,7 +163,7 @@ export default function AppSearch() {
         }}
       />
       <div className={clsx(classes.shortcut, { 'Mui-focused': focused })}>
-        {macOS ? '⌘' : 'Ctrl'}K
+        {macOS ? '⌘' : 'Ctrl'}+K
```

Comment:
On `next.material-ui.com` we only display the `+` on non-macOS systems. We've discussed this in https://github.com/mui-org/material-ui/pull/24296 (starting at https://github.com/mui-org/material-ui/pull/24296#pullrequestreview-562985225).

Do you think that we need the `+` for macOS as well (`⌘+K`) or was this mainly motivated by the display on non-macOS systems?

Reply:
I think we do want that for macOS as well, as from what I have seen it is common to write out say "Command + C".
Additionally, it also unifies the look on the two systems.

Reply:
> I think we do want that for macOS as well, as from what I have seen it is common to write out say "Command + C".

But we're not writing `Command + K` but `⌘+K`.

> Additionally, it also unifies the look on the two systems.

If your argument is about unifying then could you provide some examples where `+` is used with `⌘`? The discussion I linked has an example (VSCode) where that wasn't the case.

Reply:
Native MacOS doesn't use "+":
![image](https://user-images.githubusercontent.com/357702/104246172-6b128680-545d-11eb-9711-2b653770629e.png)

Reply:
@mbrookes Perfect, it's settles it, we shouldn't change it.